python/cudf/cudf/tests/test_string.py

```diff
@@ -1317,7 +1317,6 @@ def test_string_no_children_properties():
     assert empty_col.children == ()
     assert empty_col.size == 0
 
-    assert empty_col._nbytes == 0
```

Comment:
Safe to remove `_nbytes` and this test?

Reply:
So far as I can tell, the only place in the entire codebase using `_nbytes` (including `dask_cudf` and other packages) was this test. It seems to be cruft from an older implementation, unless it's being used somewhere completely different that's also not tested.